src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,23 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { TodoType } from '../../types/TodoType';
+import { UserInfo } from '../UserInfo';
+
+type TodoInfoType = {
+  todo: TodoType;
+};
+
+export const TodoInfo: React.FC<TodoInfoType> = ({ todo }) => {
+  const todoInfoClasses = classNames('TodoInfo', {
+    'TodoInfo--completed': todo.completed,
+  });
```

Comment:
looks super weird. Let's just move this logic to the `className` - to where it belongs